runelite-api/src/main/java/net/runelite/api/WorldType.java

```diff
@@ -131,4 +135,16 @@ public static boolean isPvpWorld(final Collection<WorldType> worldTypes)
 	{
 		return worldTypes.stream().anyMatch(PVP_WORLD_TYPES::contains);
 	}
+
+	/**
+	 * Checks whether a world having a {@link Collection} of {@link WorldType}s is a Deadman world.
+	 *
+	 * @param worldTypes A {@link Collection} of {@link WorldType}s describing the given world.
+	 * @return           True if the given worldtypes of the world are a Deadman world, false otherwise.
+	 * @see Client#getWorldType()
+	 */
+	public static boolean isDeadmanWorld(final Collection<WorldType> worldTypes)
```

Comment:
This can be replaced by `Client.getWorldType().contains(WorldType.DEADMAN)` since there is only one deadman world type.